gammapy/estimators/utils.py

```diff
@@ -461,3 +462,131 @@ def compute_lightcurve_doublingtime(lightcurve, flux_quantity="flux"):
     )
 
     return table
+
+
+def get_edges_fixed_bins(fluxpoint, value, axis_name="energy"):
+    """Rebin the flux point to combine value adjacent bins
+
+    Parameters
+    ----------
+    fluxpoint : `FluxPoints`
+        The flux point to rebin
+    value : int
+        Number of bins to combine
+
+    Returns:
+    -------
+    edges_min : `Quantity` or `Time`
+    edges_max : `Quantity` or `Time`
+        Low and High edges of the new axis
+    """
+
```

Comment:
Cant't you just use `Axis.downsample()` here?

Reply:
Downsample only works if it's divisible. However, it is a common use case to move from daily bins to weekly bins, where you might have a large number of (not a factor of 7) bins or something

Reply:
Then maybe we could change the `downsample` behavior to have a `strict=False` option which when set to `True` will simply put the remainder in the last bin as you do.

Reply:
Created an issue for this #5047